boxoffice/assets/js/views/org.js

```diff
@@ -45,9 +41,40 @@ export const OrgView = {
             errorMsg = formErrorHandler(response, this.get('formId'));
             orgComponent.set('icForm.errorMsg', errorMsg);
           });
+        }
+      });
+
+      let EditICForm = Ractive.extend({
```

Comment:
Why do the edit and new forms have to be separate components? What are the parts that are actually different?